estate/models/estate_property.py

```diff
@@ -0,0 +1,84 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models,api,exceptions
+from odoo.tools.float_utils import float_compare, float_is_zero;
+from odoo.exceptions import ValidationError;
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property"
+    _order = "id desc"
+
+    _positif_expected_price = models.Constraint("CHECK (expected_price > 0)","A price can't be negatif");
+    _positif_selling_price = models.Constraint("CHECK (selling_price > 0)","A price can't be negatif");
+
+    state = fields.Selection(selection = [("New","New"), ("Offer_Received","Offer Received") ,("Offer_Accepted","Offer Accepted"), ("Sold","Sold"), ("Cancelled","Cancelled")])
+    active = fields.Boolean('Active',default=True)
+    name = fields.Char(required=True,default="Unkown")
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date("Available From",copy=False,default= fields.Datetime.today() + relativedelta(months=3))
```

Comment:
when you run the server for the first time the default values will be stored on the db so when you open the next day it will be 3 months from day it got evaluated. but with the lambda it will be expression that will be evaluated every time 
```suggestion
    date_availability = fields.Date("Available From",copy=False,default= lambda self: fields.Datetime.today() + relativedelta(months=3))
```